Data_Structures_Questions.md

```diff
@@ -2,43 +2,59 @@ For each of the methods associated with each data structure, classify it based o
 
 ## Linked List
 
-1. What is the runtime complexity of `add_to_tail`?
+1) What is the runtime complexity of `add_to_tail`?
+- O(c), we can reference our tail property and quickly add it.
```

Comment:
I often say O(c) myself but I think it's preferred to  say O(1) either is correct though. You should be cognizant though because sometimes the constant may not actually equal 1.